pkgs/tools/misc/loop/default.nix

```
@@ -0,0 +1,24 @@
+{ stdenv, fetchFromGitHub, rustPlatform }:
+
+rustPlatform.buildRustPackage rec {
+  name = "loop-${version}";
```

The commit sha is going to be parsed as the package name:
```
nix-repl> builtins.parseDrvName "loop-d6ef3c5a0ecd4f533908abee5e481419a1a6e"
{ name = "loop-d6ef3c5a0ecd4f533908abee5e481419a1a6e"; version = ""; }
```

The nixpkgs convention is to use commit date instead (and append "unstable" to the name):
```
nix-repl> builtins.parseDrvName "loop-unstable-2018-10-02"
{ name = "loop-unstable"; version = "2018-10-02"; }
```

See nixpkgs manual: https://nixos.org/nixpkgs/manual/#sec-package-naming
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of that (fetched) commit. The date must be in "YYYY-MM-DD" format. Also append "unstable" to the name - e.g., "pkgname-unstable-2014-09-23". 


```suggestion
  name = "loop-unstable-2018-10-02";
```